src/rating/rating.component.ts

```diff
@@ -30,6 +30,8 @@ export class RatingComponent implements ControlValueAccessor, OnInit {
   @Input() readonly: boolean;
   /** array of icons titles, default: (["one", "two", "three", "four", "five"]) */
   @Input() titles: string[];
+  /** custom aria label, default "rating" */
+  @Input() ariaLabel = 'rating';
```

Comment:
please set it via config

Reply:
@valorkin I've added **Config** functionality, could you please re-check my PR?